jest.config.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
     preset: 'ts-jest',
     testEnvironment: 'node',
-    moduleNameMapper: { '@/(.*)$': '<rootDir>/src/$1' },
+    moduleNameMapper: {
+        '@/(.*)$': '<rootDir>/src/$1',
+        'config': '<rootDir>/config.js'
```

Comment:
Oh, I was soo close :D ... I just didn't understand that I had to do the change in config.js too.

Reply:
Ah :D but thanks for trying